### DISCUSSION TITLE
A Two-Factor Proof-of-Personhood (PoP) Verification/Validation Procedure: Optional Possibility to Link Idena Identities to Humanode Identities and Ensure Compliance with Strict Requirements in Projects Similar to Gitcoin Passport

### DISCUSSION THREAD

**Discussion by ltraveler** [idena-network/idena-docs#148] [General]
As far as I know, GitCoin denied the Idena request to be approved because of the possibility of having multiple accounts.

What if, for projects like GitCoin, where the "1 identity - 1 person" rule is extremely important to pass the filter control, Idena offers an additional two-factor PoP linking mechanism to the projects like Humanode.

The Humanode project has achieved an advanced level in Face ID verification (each validator have to pass the FaceID validation periodically). What if, to prove the '1 vote - 1 identity - 1 person' rule 100%, Idena Identities could link their profiles with their Humanode identities? In both networks, these linked identities would be represented by a special mark, allowing projects like Gitcoin to trust them completely and utilize them as an authentication method.

That would be an additional layer; it wouldn't be obligatory, only for people who want to use their identity in services like GitCoin Passport.

Humanode could handle the Face ID verification aspect while Idena focuses on developing the flips ceremonial part. The only question would be to ensure that Humanode and Idena ceremonies do not overlap in the same time frame.

I believe it would be a win-win strategy, constituting a two-factor Proof-of-Personhood (PoP) verification procedure. I am confident that projects like GitCoin Passport would eagerly onboard us after implementing this, enabling us to conclusively prove that identities linked to Humanode are represented by a single person.

Additionally, the collaboration between Idena and Humanode could potentially help address a significant issue faced by Humanode, known as the twins problem.

**Comment by Eurius1** [comment]
Fingerprint ID is not a good idea. Today a lot of phones (Iphone еtс) don't have a fingerprint scanner. Photo biometric authentification is much better. However there are projects like Humanode actually in place now. It would be copying I think.

**Reply by ltraveler** [reply]
I am referring more to collaboration in order to combine two projects via a Smart Contract. For example, this would make it possible to link identities between two networks and prove that, apart from the Idena Identity, you also have the humanode identity, and they are linked together.

**Comment by an-trader** [comment]
I am strongly against the provision of biometric data.
Blockchain, in my opinion, is anonymity, and providing biometric data breaks the whole point of blockchain.
In my opinion, this is the worst idea.

**Reply by ltraveler** [reply]
Please read the text one more time. There are no words related to the obligatory option of saving your private data. The idea is to provide projects like Gitcoin Passport with the opportunity to trust Idena Identities linked with identities on Humanode as an additional security layer. They can then accept Idena Identities linked to Humanode Identities as an authorization method on Gitcoin Passport.

**Reply by an-trader** [reply]
I read it carefully and understood perfectly that it is not mandatory to provide biometric data.
However, I am against even this type of identification being even voluntary.
And who, anyway, determined that Humanode is a good project?
In my opinion, projects that require biometrics are bad in any form.

**Comment by jameshille** [comment]
good
